### DISCUSSION TITLE
Any ability to get "apt" command

### DISCUSSION THREAD

**Discussion by samsibila** [skycocker/chromebrew#9831] [Q&A]
I know the package command with chromebrew is "crew" but I was wondering how, if possible, to get the apt command with the tools chromebrew provided.

sorry for for the infancy in knowledge in this I'm still learning and its a bit of an uphill battle.

**Comment by samsibila** [comment]
From my amateur research it almost looks like a good start would be to use the 'make' command to get together enough code to literally make the command but I'm very ill to the idea as i was unable to find someone who premade it.

**Reply by samsibila** [reply]
I'm also just going to ask what I think is just a blatantly dumb question if its a no, did the person who created chromium create the 'crew' command? 

I ask that because for the life of me I cant find any work/play of people using this command at all.

**Reply by uberhacker** [reply]
The `apt` command is specific to Debian-based package management.  This project, which has no association with the Chromium project or even Google, uses the `crew` command in a very similar way to `apt`.  The `dpkg` command which works with local Debian packages was added to this repo several years ago but was removed due to the conflicts it created with the filesystem directory structure of ChromeOS.  If there is a particular package that is not in this repo, I might suggest you open an issue with a request to add or try to build the package yourself.  See the [wiki](https://github.com/chromebrew/chromebrew/wiki/Creating-a-package) if you are interested.
